command/init.go

```diff
@@ -144,9 +144,70 @@ job "example" {
         }
       }
 
+      # The "artifact" stanza instructs Nomad to download an artifact from a
```

Comment:
Yea - I did most of them in this file. I'll do the rest in another PR to make it ultra-consistent.